packages/ui/src/components/templates/BottomBar/BottomBar.tsx

```diff
@@ -17,7 +17,7 @@
  */
 
 import React, {useCallback, useEffect, useMemo, useRef, useState} from 'react';
-import {Dimensions, StyleSheet, View} from 'react-native';
+import {Dimensions, SafeAreaView, StyleSheet, View} from 'react-native';
```

Comment:
Can you try to use react-native-static-safe-area-insets and see if it works please ?